src/components/AddTodo/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react'
+import CSSModules from 'react-css-modules';
+
+import styles from './styles.css';
+
+const AddTodo = ({onClick, actionId, todo}) => {
+  let text = "+"
+  if (todo) {
+    text = todo.completed ? "☑" : "☐"
+  }
+  return (
+    <button styleName='add-todo' onClick={function() {onClick(actionId)}}>
+      {text}
+    </button>
+  )
+}
+
+AddTodo.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  actionId: PropTypes.string.isRequired,
+  todo: PropTypes.object.isRequired
```

Comment:
In my console, I'm seeing the following:

```
Failed prop type: The prop `todo` is marked as required in `AddTodo`, but its value is `undefined`.
```

I'm guessing we can just remove `isRequired` and all will be well?